packages/phrases/src/locales/zh-cn.ts

```diff
@@ -47,6 +47,10 @@ const errors = {
   user: {
     username_exists: '用户名已存在。',
     email_exists: '邮箱地址已存在。',
+    phone_exists: '手机号码已存在。',
```

Comment:
we can remove this until needed. also `email_exists` can be optimized in a separate PR

Reply:
removed it for now.